server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/attachment/nonIndexableAttachment.html

```diff
@@ -0,0 +1,827 @@
e="background-color: #d04437" data-operation="color" data-color="#d04437">&nbsp;</a></li><li><a href="#" style="background-color: #f79232" data-operation="color" data-color="#f79232">&nbsp;</a></li><li><a href="#" style="background-color: #f6c342" data-operation="color" data-color="#f6c342">&nbsp;</a></li><li><a href="#" style="background-color: #654982" data-operation="color" data-color="#654982">&nbsp;</a></li><li><a href="#" style="background-color: #f691b2" data-operation="color" data-color="#f691b2">&nbsp;</a></li></ul></div></div><div id="wiki-edit-dropdown2-icon-picker-wikiEdit0" class="aui-dropdown2 aui-style-default wiki-edit-picker wiki-edit-icon-picker" data-container="wiki-edit-icon-picker-buttons-wikiEdit0"><div class="aui-dropdown2-section"><ul><li><a original-title=":)" href="#" class="wiki-edit-operation" data-operation=":)"><img src="test_files/smile.gif" alt="smile"></a></li><li><a original-title=":(" href="#" class="wiki-edit-operation" data-operation=":("><img src="test_files/sad.gif" alt="sad"></a></li><li><a original-title=":P" href="#" class="wiki-edit-operation" data-operation=":P"><img src="test_files/tongue.gif" alt="tongue"></a></li><li><a original-title=":D" href="#" class="wiki-edit-operation" data-operation=":D"><img src="test_files/biggrin.gif" alt="biggrin"></a></li><li><a original-title=";)" href="#" class="wiki-edit-operation" data-operation=";)"><img src="test_files/wink.gif" alt="wink"></a></li><li><a original-title="(y)" href="#" class="wiki-edit-operation" data-operation="(y)"><img src="test_files/thumbs_up.gif" alt="thumbs up"></a></li><li><a original-title="(n)" href="#" class="wiki-edit-operation" data-operation="(n)"><img src="test_files/thumbs_down.gif" alt="thumbs down"></a></li><li><a original-title="(i)" href="#" class="wiki-edit-operation" data-operation="(i)"><img src="test_files/information.gif" alt="information"></a></li><li><a original-title="(/)" href="#" class="wiki-edit-operation" data-operation="(/)"><img src="test_files/check.gif" alt="check"></a></li><li><a original-title="(x)" href="#" class="wiki-edit-operation" data-operation="(x)"><img src="test_files/error.gif" alt="error"></a></li><li><a original-title="(!)" href="#" class="wiki-edit-operation" data-operation="(!)"><img src="test_files/warning.gif" alt="warning"></a></li><li><a original-title="(+)" href="#" class="wiki-edit-operation" data-operation="(+)"><img src="test_files/add.gif" alt="add"></a></li><li><a original-title="(-)" href="#" class="wiki-edit-operation" data-operation="(-)"><img src="test_files/forbidden.gif" alt="forbidden"></a></li><li><a original-title="(?)" href="#" class="wiki-edit-operation" data-operation="(?)"><img src="test_files/help_16.gif" alt="help"></a></li><li><a original-title="(on)" href="#" class="wiki-edit-operation" data-operation="(on)"><img src="test_files/lightbulb_on.gif" alt="lightbulb on"></a></li><li><a original-title="(off)" href="#" class="wiki-edit-operation" data-operation="(off)"><img src=
```

Comment:
The last line here is not collored by github. Maybe the non indexable attachment is in here....